test/floki_test.exs

```diff
@@ -141,6 +146,40 @@ defmodule FlokiTest do
                    ]
       end
     end
+
+    test "parse a HTML with tags that are plain text" do
+      validate_html = fn tag ->
+        {:ok, parsed} =
+          tag
+          |> html_with_tag_that_should_not_have_children()
+          |> Floki.parse_document()
+
+        current_parser = Application.get_env(:floki, :html_parser)
+
+        case current_parser do
+          Mochiweb ->
```

Comment:
For now this is good, but I think we could do something like a `tag` for tests that are specific to a configured parser.

Reply:
Agreed! At first I intended to assert different result for different parser since html5ever was parsing new lines differently. But then html5ever stopped working properly and I got tired of debugging it so we arrived at this. Will add a `tag` with the title PR